Backpack/Map/Classes/BPKMapAnnotationView.m

```diff
@@ -159,5 +159,9 @@ - (void)traitCollectionDidChange:(UITraitCollection * _Nullable)previousTraitCol
     }
 }
 
+- (void)prepareForReuse {
```

Comment:
Shouldn't this call `[super prepareForReuse]`

Reply:
It totally should. Just pushed :D